yast/src/modules/SMTData.rb

```diff
@@ -0,0 +1,1480 @@
+# encoding: utf-8
+
+# File:	modules/SMTData.ycp
+# Package:	Configuration of SMT
+# Summary:	SMT global data and functions
+# Authors:	Lukas Ocilka <locilka@suse.cz>
+#
+# $Id:$
+require "yast"
+
+module Yast
+  class SMTDataClass < Module
+    def main
+      Yast.import "UI"
+      textdomain "smt"
+
+      Yast.import "Message"
+      Yast.import "Service"
+      Yast.import "Report"
+      Yast.import "FileUtils"
+      Yast.import "String"
+      Yast.import "Label"
+      Yast.import "Popup"
+      Yast.import "PackageSystem"
+      Yast.import "Wizard"
+      Yast.import "Progress"
+      Yast.import "Directory"
+      Yast.import "ProductControl"
+
+      @all_credentials = {}
+
+      @smt_conf = "/etc/smt.conf"
+
+      # script entered into a crontab
+      @cron_mirror_script = "/usr/bin/smt-mirror.pl"
+      # cron
+      @path_to_cron = "/etc/smt.d/novell.com-smt"
+
+      @path_to_ncc_sync_script = "/usr/sbin/smt-ncc-sync"
```

Comment:
Correct me if I'm wrong, but from one IRC conversation I understood this script should no longer be in SMT12. It (and some others) depends on `SMT::NCCRegTools` which was removed. I also deleted this file in my dev branch.